v2/cmd/generate.go

```diff
@@ -198,7 +201,8 @@ func init() {
 	generateCmd.Flags().BoolVar(&generateCmdOpts.FromDDL, "from-ddl", false, "toggle using ddl file")
 	generateCmd.Flags().StringVarP(&generateCmdOpts.Out, "out", "o", "", "output path or file name")
 	generateCmd.Flags().StringVar(&generateCmdOpts.Suffix, "suffix", defaultSuffix, "output file suffix")
-	generateCmd.Flags().StringVarP(&generateCmdOpts.Package, "package", "p", "", "package name used in generated Go code")
+	generateCmd.Flags().StringVarP(&generateCmdOpts.PackagePath, "package-path", "p", "", "Go package import path for the generated Go code")
```

Comment:
We need to know the full package path to tell if a given package is local or not, so I added a new flag `package-path`, which is required.